integration-tests/prod_test_constant.py

```diff
@@ -0,0 +1,130 @@
+                  "product-ei": {"WSO2_CARBON_DB_CORE": ['dbscripts/mysql5.7.sql'],
+                                 "WSO2_CARBON_DB_BROKER": ['wso2/broker/dbscripts/mysql5.7.sql'],
+                                 "WSO2_CARBON_DB_BPS": ['wso2/business-process/dbscripts/mysql5.7.sql'],
+                                 "WSO2_MB_STORE_DB_BROKER": ['wso2/broker/dbscripts/mb-store/mysql-mb.sql'],
+                                 "WSO2_METRICS_DB_BROKER": ['wso2/broker/dbscripts/metrics/mysql.sql'],
+                                 "BPS_DS_BPS": ['wso2/business-process/dbscripts/bps/bpel/create/mysql.sql'],
+                                 "ACTIVITI_DB_BPS": [
+                                     'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mysql.create.engine.sql',
+                                     'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mysql.create.history.sql',
+                                     'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mysql.create.identity.sql',
+                                     'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mysql.create.substitute.sql']}}},
+
+    "SQLSERVER-SE": {"prefix": "jdbc:sqlserver://",
+                     "driverClassName": "com.microsoft.sqlserver.jdbc.SQLServerDriver", "jarName": "sqlserver-ex.jar",
+                     "DB_SETUP": {
+                         "product-apim": {},
+                         "product-is": {},
+                         "product-ei": {"WSO2_CARBON_DB_CORE": ['dbscripts/mssql.sql'],
+                                        "WSO2_CARBON_DB_BROKER": ['wso2/broker/dbscripts/mssql.sql'],
+                                        "WSO2_CARBON_DB_BPS": ['wso2/business-process/dbscripts/mssql.sql'],
+                                        "WSO2_MB_STORE_DB_BROKER": ['wso2/broker/dbscripts/mb-store/mssql-mb.sql'],
+                                        "WSO2_METRICS_DB_BROKER": ['wso2/broker/dbscripts/metrics/mssql.sql'],
+                                        "BPS_DS_BPS": ['wso2/business-process/dbscripts/bps/bpel/create/mssql.sql'],
+                                        "ACTIVITI_DB_BPS": [
+                                            'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mssql.create.engine.sql',
+                                            'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mssql.create.history.sql',
+                                            'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mssql.create.identity.sql',
+                                            'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mssql.create.substitute.sql']}}},
+
+    "ORACLE-SE2": {"prefix": "jdbc:oracle:thin:@", "driverClassName": "oracle.jdbc.OracleDriver",
+                   "jarName": "oracle-se.jar",
+                   "DB_SETUP": {
+                       "product-apim": {},
+                       "product-is": {},
```

Comment:
Why are these there? shall we remove please.

Reply:
Removed product ids and fixed in 5b5b94186229d940cfe147d6a2a8071b0d0320de